app/main/routes.py

```diff
@@ -84,14 +84,57 @@ def callback():
     session["token_scope"] = access_token_response["scope"]
     session["session_state"] = access_token_response["session_state"]
 
-    return redirect(url_for("main.poc_search"))
+    return redirect(url_for("main.browse"))
 
 
 @bp.route("/accessibility", methods=["GET"])
 def accessibility():
     return render_template("accessibility.html")
 
 
+@bp.route("/browse", methods=["POST", "GET"])
+@access_token_sign_in_required
+def browse():
+    transferring_body_id = request.args.get("transferring_body_id", None)
+    series_id = request.args.get("series_id", None)
+    consignment_id = request.args.get("consignment_id", None)
+
+    form = SearchForm()
+    browse_results = []
+    browse_type = "browse"
+
+    # Create a list to accumulate filter conditions
+    filters = []
+
+    if transferring_body_id:
+        filters.append({"transferring_body_id": transferring_body_id})
+        browse_type = "transferring_body"
+    if series_id:
+        filters.append({"series_id": series_id})
+        browse_type = "series"
+    if consignment_id:
+        filters.append({"consignment_id": consignment_id})
+        browse_type = "consignment"
+
+    # Check browse type
+    if browse_type == "browse":
```

Comment:
I think this filter can be simplified from 
`browse_data(**{k: v for d in filters for k, v in d.items()})` to `browse_data(**filters)`

if we adjust the filters object to be a dict rather than list of dict, i.e

```
filters = {}
if transferring_body_id:
    filters["transferring_body_id] = transferring_body_id
....
```

And further to that I think we could get away with just rather than needing this if else block:
`browse_results = browse_data(**{k: v for d in filters for k, v in d.items()})`
but not a big deal

Reply:
I think, filter list /dictionary won't be required when we integrate as being passed as individual parameters to browse_data query

Reply:
This will be changed to account for single queries which should be good for browse. When it comes to filter functionality we may want to consider compounding filters.